testsuite/rotational-diffusion-aniso.py

```diff
@@ -0,0 +1,326 @@
+from __future__ import print_function
+import unittest as ut
+import numpy as np
+from numpy.random import random, seed
+import espressomd
+import math
+import tests_common as tc
+
+@ut.skipIf(not espressomd.has_features(["ROTATION",
+                                        "PARTICLE_ANISOTROPY",
+                                        "ROTATIONAL_INERTIA",
+                                        "DIPOLES"]),
+           "Features not available, skipping test!")
+class RotDiffAniso(ut.TestCase):
+    longMessage = True
+    # Handle for espresso system
+    es = espressomd.System(box_l=[1.0,1.0,1.0])
+    es.cell_system.skin = 5.0
+    es.seed = 5
+    
+    # The NVT thermostat parameters
+    kT = 0.0
+    gamma_global = np.zeros((3))
+    
+    # Particle properties
+    J = 0.0,0.0,0.0
+    
+    @classmethod
+    def setUpClass(cls):
+        np.random.seed(4)
+
+    def setUp(self):
+        self.es.time = 0.0
+        self.es.part.clear()
+        print("\n")
+
+    def generate_scalar_ranged_rnd(self, min_par, max_par):
```

Comment:
you don't need those two functions if you use https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.uniform.html
and if this does not serve your needs (but I think it does), you can use the shape as an argument